### DISCUSSION TITLE
2005/11/14/the-%E2%80%9Conly-child%E2%80%9D-generation

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#57] [Announcements]
# 2005/11/14/the-%E2%80%9Conly-child%E2%80%9D-generation

The majority of Chinese families now have only one child, thanks to the one-child policy that has been in place for nearly 20 years. By 2001, there were already 240 million “only children” in China—a population roughly equivalent to that of the United States. In the coming decades, this generation of “only children” will rise to become the elite of society, and their influence on the world remains to be seen.

https://blog.qizhen.xyz/2005/11/14/the-%E2%80%9Conly-child%E2%80%9D-generation

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Terry
2008年10月12日 13:19 编辑
写得不错哦 我准备IELTS topic找到这了
哈哈

回复
wei
2007年10月19日 09:17 编辑
sorry not "could",it\’s "called"

回复
wei
2007年10月19日 09:17 编辑
I think it should also be could "Birth control", most newspaper such as New York Times said so.

回复
庆君
2007年09月3日 18:52 编辑
写得不错啊！
文笔很流畅的
